themes/default/content/blog/pulumi-release-notes-m57/index.md

```diff
@@ -0,0 +1,165 @@
+
+Learn how to get started with [ECS Anywhere]({{< relref "/blog/ecs-anywhere-launch" >}}) and [App Runner]({{< relref "/blog/deploy-applications-with-aws-app-runner" >}}).
+
+### Pulumi supports Sumo Logic
+
+The new Sumo Logic provider for Pulumi allows you to manage Sumo Logic's big data and analytics services with Pulumi infrastructure-as-code.
+
+{{< chooser language "javascript,typescript,python,go,csharp" >}}
+
+{{% choosable language javascript %}}
+
+```javascript
+const sumologic = require("@pulumi/sumologic")
+
+const folder = new sumologic.Folder("folder", {
+  description: "A test folder",
+  parentId: "<personal folder id>",
+});
+```
+
+{{% /choosable %}}
+{{% choosable language typescript %}}
+
+```typescript
+import * as sumologic from "@pulumi/sumologic";
+
+const folder = new sumologic.Folder("folder", {
+  description: "A test folder",
+  parentId: "<personal folder id>",
+});
+
+```
+
+{{% /choosable %}}
+{{% choosable language python %}}
+
+```python
+import pulumi_sumologic as sumologic
+
+folder = sumologic.Folder("folder",
+                          description="A test folder",
+                          parent_id="<personal folder id>")
+```
+
+{{% /choosable %}}
+{{% choosable language go %}}
+
+```go
+import (
+	"github.com/pulumi/pulumi/sdk/v3/go/pulumi"
+	"github.com/pulumi/pulumi-sumologic/sdk/go/sumologic"
+)
+
+func main() {
+	pulumi.Run(func(ctx *pulumi.Context) error {
+		_, err := sumologic.NewFolder(ctx, "folder", &sumologic.FolderArgs{
+			Description: pulumi.String("A test folder"),
+            ParentId:    pulumi.String("<personal folder id>"),
+		})
+		if err != nil {
+			return err
+		}
+
+		return nil
+	})
+}
+```
+
+{{% /choosable %}}
+{{% choosable language csharp %}}
+
+```csharp
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Pulumi;
+using Pulumi.SumoLogic;
+
+class Program
+{
+    static Task Main() =>
+        Deployment.Run(() => {
+            var folder = new SumoLogic.Folder("folder", new SumoLogic.FolderArgs
+            {
+                Description = "A test folder",
+                 ParentId = "<personal folder id>",
+            });
+        });
+}
+```
+
+{{% /choosable %}}
+
+{{< /chooser >}}
+
+Get started in the [SumoLogic provider docs]({{< relref "/docs/intro/cloud-providers/sumologic" >}}).
+
+### 100 new resources in the Azure Native provider
+
+We shipped 3 new versions of the Azure Native provider (1.10.0, 1.9.0, and 1.8.0) that collectively added 100 new resources that you can manage with the Azure Native provider. These new provider versions give you full access to the new resources and services announced at [Microsoft Build 2021](https://news.microsoft.com/build-2021-book-of-news/), including new analytics, machine learning, and security capabilities across the Azure platform.
+
+[See the full list](https://github.com/pulumi/pulumi-azure-native/blob/v1.10.0/CHANGELOG.md)
+
+## Pulumi engine and core technologies
```

Comment:
should we just call this the Pulumi CLI and core technologies?

Reply:
```suggestion
## Pulumi CLI and core technologies
```